src/crypto/wincrypto/cert.rs

```diff
@@ -0,0 +1,38 @@
+use crate::crypto::dtls::DTLS_CERT_IDENTITY;
+use crate::crypto::Fingerprint;
+use std::sync::Arc;
+use str0m_wincrypto::{WinCryptoCertificate, WinCryptoError};
+
+#[derive(Clone, Debug)]
+pub struct WinCryptoDtlsCert {
+    pub(crate) certificate: Arc<WinCryptoCertificate>,
+}
```

Comment:
The distinction between `WinCryptoDtlsCert` and `WinCryptoCertificate` is a bit dodgy here. One is a local type, the other is in a crate. I think we should figure out a naming that makes this clearer.

One way would be to rename the crate types so that `WinCryptoCertificate` -> `Certificate` and consistently use the crate types with a crate prefix `wincrypto::Certificate`. That takes some discipline though since code tools like VSCode will auto-import for you.

You often see this pattern in other projects like `io::Error` etc. In ureq I decided the `http` crate is always used as `http::Request` for instance.

Reply:
Ok.. so preference would be that the `str0m-wincrypto` crate types don't prefix their public types/functions .. and instead we rely on using `str0m_wincrypto::` to be explicit where the types come from. `WinCryptoError` would be the exception since in the str0m_wincrypto crate it'll be a mess to refer to `crate::Error`

Reply:
Yep. That all sounds right.